test/simulator/timestamp/src/include/timestamp_manager.h

```diff
@@ -41,14 +41,14 @@ class timestamp_manager {
     void parse_config(const std::string &, std::map<std::string, std::string> &);
```

Comment:
As discussed, the `parse_config` function could be improved and handle all the checks related to the configuration. Once this function has been called, we should not check if the config is empty, valid or not, we should have the guarantee it is.
We could give it a vector of "accepted" values and a vector of "unsupported values" (not sure about this, it needs more thoughts) and the function would generate the map with key/value pairs that are valid.

Reply:
Updated in [ceae83e](https://github.com/wiredtiger/wiredtiger/pull/8372/commits/ceae83e772527ea5fd3bb258cf943e372153a5e3)